packages/babel-helper-transform-fixture-test-runner/src/index.js

```diff
@@ -172,6 +322,7 @@ function run(task) {
   if (execCode) {
     const execOpts = getOpts(exec);
     result = babel.transform(execCode, execOpts);
+    checkDuplicatedNodes(result.ast);
```

Comment:
i guess this might create quite some overhead, ideally we should not do this by default, maybe only on CI?

Reply:
I will disable it on normal `make test` at the end of the pr, when the tests are green.

Reply:
is it actually slower?

Reply:
With the check:
```
make test-only  60,29s user 4,13s system 115% cpu 55,727 total
```

Without:
```
make test-only  57,88s user 3,98s system 115% cpu 53,697 total
```

We can always run the check.